src/components/AuthPage/ForgotPassword/index.jsx

```diff
@@ -43,7 +43,13 @@ const ForgotPassword = ({
 	const loadingProps = useSelector(({ auth }) => auth.profile.loading);
 	const dispatch = useDispatch();
 
-	useEffect(() => setError(errorProps), [errorProps]);
+	useEffect(() => {
+		if (errorProps === "Firebase: Error (auth/missing-email).") {
```

Comment:
@shivareddy6 add the error case at `src/helpers/errorMsgHandler.jsx` instead of here

Reply:
sure